test/metabase/test_runner/effects.clj

```diff
@@ -45,10 +45,22 @@
        :diffs    (when-not pass?#
                    [[actual# [(s/check schema# actual#) nil]]])})))
 
+(defn derecordize
```

Comment:
wait, where's the side effect? this one looks pure?

Reply:
You're right. This namespace doesn't actually have any side effects anymore. It did at one point but all that stuff was moved out a while ago. I went ahead and renamed the namespace to reflect that.